src/login/LoginFailure.jsx

```diff
@@ -29,7 +29,7 @@ const LoginFailureMessage = (props) => {
   const { formatMessage } = useIntl();
   const authService = getAuthService();
   const {
-    context, errorCode, failureCount,
+    context, errorCode, failureCount, tpaContext,
```

Comment:
Can we please use the `context` prop and pass any error message inside it. This component should not worry about from where the error message is coming. right?

Reply:
addressed by creating new function inside the LoginPage component which is return the error context and removed the tpaContext from Failure component